consensus/core/src/metrics.rs

```diff
@@ -800,3 +1179,813 @@ impl NodeMetrics {
         }
     }
 }
+
+pub(crate) struct ValidatorScoreMetrics {
```

Comment:
I would rename this to `ScoringMetrics` to match the variable name, and everywhere we have a name that ends in `ScoreMetrics` I would rename to `ScoringMetrics`. `Score` will then only be used in names that relate to the actual validator score which combines the scoring metrics.

Reply:
yeah, you are right, let me change that.